semantic/src/site/modules/modal.variables

```diff
@@ -1,6 +1,3 @@
 /*******************************
          Site Overrides
 *******************************/
-
-// Prevent rounded corners of full-screen modal
```

Comment:
This should only be applied to full-screen modals, so have moved to "modal.overrides" and applied to correct class